tests/functional/config.py

```diff
@@ -66,6 +63,13 @@
 """ TT_TARGET_DEVICE - one of "CPU", "GPU" """
 target_device = os.environ.get("TT_TARGET_DEVICE", "CPU")
 
+"""IMAGE - docker image name which should be used to run tests"""
+if target_device == "CPU":
```

Comment:
in case of HDDL & NCS we would like to use the default Image (non-GPU)

Reply:
Done